core/src/main/java/org/elasticsearch/node/service/NodeService.java

```diff
@@ -150,7 +151,8 @@ public NodeInfo info(boolean settings, boolean os, boolean process, boolean jvm,
                 transport ? transportService.info() : null,
                 http ? (httpServer == null ? null : httpServer.info()) : null,
                 plugin ? (pluginService == null ? null : pluginService.info()) : null,
-                ingest ? (ingestService == null ? null : ingestService.info()) : null
+                ingest ? (ingestService == null ? null : ingestService.info()) : null,
+                indicesService == null ? null : indicesService.getTotalIndexingBufferBytes()
```

Comment:
should we add a new "indices" metric (See RestNodesInfoAction), which will correspond to a new boolean in this class?